rosdep/python.yaml

```diff
@@ -1553,6 +1553,10 @@ python-pytest:
   debian: [python-pytest]
   fedora: [python-pytest]
   ubuntu: [python-pytest]
+python-pytz:
+  ubuntu:
+    pip:
+      packages: [pytz]
```

Comment:
Can you please use the convention where you add `-pip` to the end of the key, since you're specifying a pip dependency on Ubuntu. The reason is so that if a Ubuntu package for `pytz` exists or becomes available the `python-pytz` would be available for it to use.

Thanks.